src/frontend/komponenter/Fagsak/Behandlingsresultat/useHentEndretUtbetalingBegrunnelser.tsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+import React from 'react';
+
+import { useHttp } from '@navikt/familie-http';
+import { byggTomRessurs, type Ressurs } from '@navikt/familie-typer';
+
+import type { EndringsårsakbegrunnelseTekster } from '../../../typer/endretUtbetaling';
+
+export function useHentEndretUtbetalingBegrunnelser() {
+    const { request } = useHttp();
+
+    const [endretUtbetalingsbegrunnelser, settEndretUtbetalingsbegrunnelser] =
+        React.useState<Ressurs<EndringsårsakbegrunnelseTekster>>(byggTomRessurs());
+
+    useEffect(() => {
+        request<void, EndringsårsakbegrunnelseTekster>({
+            method: 'GET',
+            url: `/familie-ks-sak/api/endretutbetalingandel/endret-utbetaling-vedtaksbegrunnelser`,
+            påvirkerSystemLaster: true,
+        }).then((data: Ressurs<EndringsårsakbegrunnelseTekster>) => {
+            settEndretUtbetalingsbegrunnelser(data);
+        });
+    }, []);
```

Comment:
Burde denne ta inn årsak og kun hente begrunnelser når årsak endres og er satt til `ALLEREDE_UTBETALT`? Typ:
```suggestion
    useEffect(() => {
        if(årsak == IEndretUtbetalingAndelÅrsak.ALLEREDE_UTBETALT) {
            request<void, EndringsårsakbegrunnelseTekster>({
                method: 'GET',
                url: `/familie-ks-sak/api/endretutbetalingandel/endret-utbetaling-vedtaksbegrunnelser`,
                påvirkerSystemLaster: true,
            }).then((data: Ressurs<EndringsårsakbegrunnelseTekster>) => {
                settEndretUtbetalingsbegrunnelser(data);
            });
        }
    }, [årsak]);
```

Reply:
Enig! Jeg kan fikse det ☺️

Reply:
Jeg sjekket funksjonaliteten og ser at denne hooken kun blir kalt når avslagsbegrunnelser skal vises. Det vil si at den ikke trenger noe ekstra logikk for å kun trigge når årsaken er ALLEREDE_UTBETALT. 😊